udnssdk.go

```diff
@@ -64,17 +91,19 @@ func NewClient(username, password, BaseURL string) (*Client, error) {
 // {"tokenType":"Bearer","refreshToken":"48472efcdce044c8850ee6a395c74a7872932c7112","accessToken":"b91d037c75934fc89a9f43fe4a","expiresIn":"3600"
 // ,"expires_in":"3600","token_type":"Bearer","refresh_token":"48472efcdce044c8850ee6a395c74a7872932c7112","access_token":"b91d037c75934fc89a9f43fe4a"}
 
+// AuthResponse wraps the response to an auth request
 type AuthResponse struct {
 	TokenType     string `json:"tokenType"`
 	RefreshToken  string `json:"refreshToken"`
 	AccessToken   string `json:"accessToken"`
 	ExpiresIn     string `json:"expiresIn"`
-	Expires_in    string `json:"expires_in"`
-	Token_type    string `json:"token_type"`
-	Refresh_token string `json:"refresh_token"`
-	Access_token  string `json:"access_token"`
+	Expires_in    string `json:"expires_in"`     // yes, these have terrible names.
```

Comment:
is `expires_in` always going to contain the same value as `expiresIn`? Can we just ignore these attributes from the response?